components/TodoList.tsx

```diff
@@ -25,11 +25,11 @@ const ListTodos = observer(function ListTodos() {
           key={todo.id}
           value={todo.id}
           borderColor="black"
-          borderWidth="0.1em"
+          borderWidth={1}
         >
           <Accordion.Header>
             <XStack alignContent="center">
-              <Text paddingLeft="0.5em">{todo.title}</Text>
+              <Text paddingLeft={20}>{todo.title}</Text>
```

Comment:
I think what you need here are the following:
1) Make sure that XStack is full width
2) Make Text using Flex=1, so that it grows as long as possible
but still leaving space for the buttons on the right